### PR TITLE
Configure GitHub workflow for ignition-manifests-and-kubeconfig-generate

### PR DESCRIPTION
**Pull request by eranco74**
Added build using skipper
Added pep and pylint
Added  git action build on PR and master push.
Updated the release workflow

### CODE REVIEW

**Comment by eranco74** [comment]
Tested full install :heavy_check_mark:

File: render_files.py [hunk 1]
```diff
@@ -38,43 +35,43 @@ def upload_to_aws(s3, local_file, bucket, s3_file):
...
 
     for root, _, files in os.walk(install_dir):
-        for f in files:
-            logging.info("Uplading file: {}".format(f))
-            file_path = os.path.join(root, f)
-            if f == "kubeconfig":
-                f = "kubeconfig-noingress"
-            s3_file_name = "{}/{}".format(prefix, f)
+        for file_name in files:
+            logging.info("Uplading file: %s", file_name)
```
Comment (yevgeny-shnaidman) [inline render_files.py]: Typo:  uploading

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,14 @@
+SERVICE := $(or ${SERVICE},quay.io/ocpmetal/ignition-manifests-and-kubeconfig-generate:latest)
+
+
 all: pep8 pylint build
 
 build:
 	skipper build ignition-manifests-and-kubeconfig-generate
 
+update: build
```
Comment (yevgeny-shnaidman) [inline Makefile:9]: I, personally against target that builds and pushes. I think it may lead to people pushing, by mistake, their debug images into latest

**Review by yevgeny-shnaidman — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (eranco74) [reply Makefile:9]: Agree, Although people shouldn't be login to ocpmetal...

**Pull request merged**